src/elements/_lists.pug

```diff
@@ -27,9 +27,9 @@ mixin list--block__parties(party, count)
       .card__details
         h2 #[+link('#', party)]
         p= 'All current ' + party + ' MPs'
-      .card__details
-        span.count= count
-        | MPs
+      .count
+        span.count--number= count
+        span.count--context MPs
```

Comment:
Following BEM principles like our card component styles. Should the number and context look instead like this; with double underscores rather than double dashes?

```
.card
  .card__number
  .card__details
```

Reply:
FYI: I'm not requesting you to change this; it's just a suggestion.